pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -171,6 +171,9 @@
             + " affecting the accuracy of the delivery time compared to the scheduled time. Default is 1 second.")
     private long delayedDeliveryTickTimeMillis = 1000;
 
+    @FieldContext(category = CATEGORY_SERVER, doc = "Whether to enable the acknowledge of batch local index")
+    private boolean batchIndexAcknowledgeEnable = true;
```

Comment:
If this is a feature has to enabled on the broker, is there a way for consumer to know or set such feature is supported? I think the problem comes how to help troubleshooting when both broker and consumer need to support this.

Reply:
Should we consider adding a flag to enable or disable at consumer side? I think this feature does not introduce too much overhead on the consumer side.

For troubleshooting, users just to ensure that their brokers are enabled this feature and the client version is 2.6.0